core/templates/pages/exploration-editor-page/exploration-editor-page.component.html

```diff
@@ -34,43 +34,43 @@
         </div>
       </li>
 
-      <li ng-class="{'navbar-tab-active': $ctrl.getActiveTabName() === 'translation'}" id="tutorialTranslationTab" class="nav-item icon nav-list-item protractor-test-translation-tab" ng-click="$ctrl.selectTranslationTab()">
+      <li ng-class="{'navbar-tab-active': $ctrl.getActiveTabName() === 'translation', 'oppia-disabled-link': !$ctrl.connectedToInternet}" id="tutorialTranslationTab" class="nav-item icon nav-list-item protractor-test-translation-tab" ng-click="$ctrl.selectTranslationTab()">
```

Comment:
For disabled tabs, could we show a tooltip to let the user know that the functionality is disabled because there is no connectivity?

Reply:
Done. Thanks!